AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -335,7 +336,9 @@ - (void)endUpdatesWithCompletion:(void (^)(BOOL))completion
   _batchUpdateCounter--;
 
   if (_batchUpdateCounter == 0) {
-    [_delegate dataControllerBeginUpdates:self];
+    [_editingTransactionQueue addOperationWithBlock:^{
+      [_delegate dataControllerBeginUpdates:self];
```

Comment:
Ahh yes I see that now. I got a little confused because ASRangeController also re-wraps it's delegate calls in a main thread block so I thought ASDataControllerDelegate was a background thing. Fixed.